unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -29,13 +29,13 @@
 
   let credential = $state.credentials.find((c) => $page.params.id === c.id)!!;
 
-  //   let color = credential.metadata.display.color || colors.at(0);
-
   let icon: any = credential.metadata.display.icon || 'User';
   let title: string = credential.metadata.display.name || credential.data.type.at(-1);
 
   let credentialLogoUrl: string | null;
-  let issuerLogoUrl: string | null;
+
+  // TODO: add issuer id which can then be used to link to the connection
+  let issuerId: string | undefined;
```

Comment:
let issuerId?: string;

Reply:
I removed it for now since we cannot use it just yet (linking to connections).